tests/composition_cg_dg_fem.prm

```diff
@@ -0,0 +1,95 @@
+# A test to check discontinuous composition discretization.
```

Comment:
Description needs an update

Reply:
If you copied the test from some other test, can you include that test and only list the lines that are different?

Reply:
is this really what we do? wouldn't that create annoying dependencies between now unrelated tests?